homeassistant/components/mastodon/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: |
+      Legacy Notify platform can't be created within async-setup.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow:
+    status: done
+    comment: |
+      YAML flow deprecated and will be removed in 2025.2.0
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration do not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      Legacy Notify platform is not an action.
```

Comment:
Yes and no, it is something that can be executed and can raise

Reply:
Marking as todo since this is the legacy notify platform and I want to rip it all out next year and go for gold.